updates for Testkube OpenSource (#22)
kubeshop/testkube-docs: docs/articles/logs-and-artifacts.md

# Logs & Artifacts

## Log Collection

Testkube automatically collects logs from containers running tests within a Workflow and makes these available in

> the Dashboard when open an execution and selecting the "Log Output" tab (which is default):

Reviewer: ```suggestion
the Dashboard when open an execution and selecting the "Log Output" tab (which is the default):
```

![Workflow Log output](../img/workflows-log-output.png)

Logs are collected individually for each executed container within a workflow, and can thus be inspected separately:

![Workflow Multi-log output](../img/workflows-multilog-output.png)

## Artifact Collection

> Workflows that produce artifact can be configured to upload those to Testkubes Artifact storage - read more. These 

Reviewer: ```suggestion
Workflows that produce artifacts can be configured to upload those to Testkube's Artifact storage - read more. These 
```

Reviewer: Should "read more" be a link?

> could for example be videos, logs, html-reports, JUnit XML reports, etc. - anything that you want to make available

Reviewer: ```suggestion
could be, for example, videos, logs, html-reports, JUnit XML reports, etc. - anything that you want to make available
```

> for inspecting / troubleshooting a test execution. 

Reviewer: ```suggestion
for inspecting or troubleshooting a test execution. 
```
